README.md

```diff
@@ -4,6 +4,7 @@
 </p>
 
 <p align='center'>
+
```

Comment:
Looks like it does help:
![react-component-queries readme md at 3af28b0ccf24872b454de54c593c34427e7e801d oyeanuj react-component-queries 2017-05-31 22-50-18](https://cloud.githubusercontent.com/assets/223620/26650892/94aaa14c-4653-11e7-8020-bdb3c7fa1026.png)